src/lib/urn.ts

```diff
@@ -122,6 +118,9 @@ export function isThirdParty(urn?: string) {
 }
 
 export function decodeURN(urn: URN): DecodedURN {
+  const urnRegExp = new RegExp(
```

Comment:
Moved this const `urnRegExp ` inside the method because `Safari` && `Safari on iOS` doesn't support `lookbehind` in JS regular expressions.